spec/api-web-contents-spec.js

```diff
@@ -347,6 +347,54 @@ describe('webContents module', () => {
     })
   })
 
+  describe('zoom-changed', () => {
+    before(function () {
+      // On Mac, zooming isn't done with the mouse wheel.
+      if (process.platform === 'darwin') {
+        this.skip()
+      }
+    })
+
+    it('is emitted with the correct zooming info', (done) => {
+      w.loadURL(`file://${path.join(__dirname, 'fixtures', 'pages', 'base-page.html')}`)
```

Comment:
@ppontes can you please use `loadFile` instead?

Reply:
👍🏻